### PR TITLE
[FEATURE/GUIDE] Create Guide

### PR DESCRIPTION
**Pull request by NguyenTriBaoThang**
## Description

Create guide page 

## Project

- [x] Website
- [ ] API
- [ ] Documentation
- [ ] Other... (please describe)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Other... (please describe)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B
- [x] No tests needed

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Other... (please describe)

### CODE REVIEW

File: apps/website/src/features/Guide/components/GuideVideo.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          <Grid container spacing={2} className="mb-10" key={item.id}>
+            <Grid container item xs={12} sm={6}>
+              <Box justifyContent="space-between">
+                <div className="flex items-center">
+                  <div className="w-16 h-16 pt-5 mx-3 mb-3 text-xl text-center text-white rounded-full bg-japonica-600">
+                    {item.number}
+                  </div>
+                  <div className="text-xl font-bold">{item.name}</div>
+                </div>
+                {item.content.split('\n').map((line) => (
```
Comment (qoucname2202) [inline apps/website/src/features/Guide/components/GuideVideo.tsx:18]: @NguyenTriBaoThang Write it as a separate function and call the function inside

File: apps/website/src/features/Guide/components/GuideVideo.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            </Grid>
+            <Grid container item xs={12} sm={6}>
+              <Box justifyContent="space-between">
+                <div className="flex items-center">
+                  <div className="w-16 h-16 pt-5 mx-3 mb-3 text-xl text-center text-white rounded-full bg-japonica-600">
+                    {item.number}
+                  </div>
+                  <div className="text-xl font-bold">{item.name}</div>
+                </div>
+                {item.content.split('\n').map((line) => (
```
Comment (qoucname2202) [inline apps/website/src/features/Guide/components/GuideVideo.tsx:63]: @NguyenTriBaoThang Write it as a separate function and call the function inside

**Review by qoucname2202 — CHANGES_REQUESTED** [review]
@NguyenTriBaoThang Fix code style for each file has comment

**Review by foxminchan — CHANGES_REQUESTED** [review]
@NguyenTriBaoThang reduce duplication in the code [here](https://sonarcloud.io/component_measures?id=foxminchan_GovermentLLM&pullRequest=59&metric=new_duplicated_lines_density&view=list)

**Review by foxminchan — APPROVED** [review]

**Pull request merged**